packages/server/lib/util/ci_provider.js

```diff
@@ -468,7 +493,7 @@ const commitDefaults = function (existingInfo) {
   // defaulting back to null if all fails
   // NOTE: only properties defined in "existingInfo" will be returned
   const combined = _.transform(existingInfo, (memo, value, key) => {
-    return memo[key] = _.defaultTo(value != null ? value : commitParamsObj[key], null)
+    return memo[key] = _.defaultTo(value || commitParamsObj[key], null)
```

Comment:
hmm, changing the logic here, right? because now it treats empty strings and undefined values differently. But I don't see any changed unit tests

Reply:
This is a decaffeinate fix, just a shorter null check. It *should* be equivalent, but also, they always say "may not be equivalent", so yah I will add a unit test. 

https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md#ds207-consider-shorter-variations-of-null-checks

Reply:
ok